dev/com.ibm.ws.jpa.tests.spec10.injection.defaultPersistenceUnit_fat/RepeatWithJPA31.java

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Remove this file.... looks like a copy of an existing file into the wrong location.